packages/devtools-reps/src/reps/stubs/accessible.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+const stubs = new Map();
+stubs.set("Document", {
+  actor: "server1.conn1.child1/accessible31",
+  typeName: "accessible",
+  preview: {
+    name: "New Tab",
+    role: "document",
+    isConnected: true
+  }
+});
+
+stubs.set("ButtonMenu", {
+  actor: "server1.conn1.child1/accessible38",
+  typeName: "accessible",
+  preview: {
+    name: "New to Nightly? Let’s get started.",
+    role: "buttonmenu",
+    isConnected: true
+  }
+});
+
+stubs.set("NoName", {
+  actor: "server1.conn1.child1/accessible93",
+  typeName: "accessible",
+  preview: {
+    name: null,
+    role: "text container",
+    isConnected: true
+  }
+});
+
+stubs.set("DisconnectedAccessible", {
+  actor: null,
+  typeName: "accessible",
+  preview: {
+    name: null,
+    role: "section",
+    isConnected: false
+  }
+});
+
+const name = "a".repeat(1000);
```

Comment:
can accessible have an actual longString name (i.e. if the name length is > 9999, in the majority of object we don't send the whole string, but only it's first N char - See https://github.com/devtools-html/debugger.html/blob/e7f212ee3916d97c799549b6b89cb43fbc1ce34f/packages/devtools-reps/src/reps/stubs/element-node.js#L224-L229)

Reply:
It's quite possible. Platform a11y creates accessible objects for most text nodes. So if we have a text node that has more than 9999 chars, which is pretty reasonable, then it is the case.